checkov/sca_image/runner.py

```diff
@@ -174,11 +174,12 @@ def iterate_image_files(self, abs_fname: str, report: Report, runner_filter: Run
             if image_referencer.is_workflow_file(abs_fname):
                 images = image_referencer.get_images(file_path=abs_fname)
                 for image in images:
-                    if not strtobool(os.getenv('CHECKOV_PRESENT_CACHED_RESULTS', "False")):
+                    if strtobool(os.getenv('CKV_CREATE_SCA_IMAGE_REPORTS_FOR_IR', "True")):
```

Comment:
can you change the prefix for both env vars to `CHECKOV_...` we try to use the full name for internal, not really meant to be changed by a user and `CKV_...` is used for user related flags and behaviour changes.